### ISSUE TITLE
Costumize Background

### ISSUE THREAD

**Issue by cameronsutter** [cameronsutter/plottr_electron#202]
Submitter: Anna Fritsche
Email: anna.fritsche96@gmail.com
Hey, love this program. It looks nice and modern and is indeed very useful to keeo track of everything happening. The very light look is making it very unpleasant for the eyes over time tho. Would be nice to be able to have a custom background.

**Comment by cameronsutter** [comment]
More details from Anna:

a) A simple night mode: I've seen this in several programs or websites already. An on/off switch which the user can activate night mode with. All of the White elements would become a soft grey. All grey elements a darker grey and so on. 

b) A slightly transparent work space: I'm guessing this one's going to be more complicated. It would mean that the user could choose his own wallpaper. All of the different panels would still be able to be seen by the different transparency. I hope you know what I mean.
I have seen a mixture between a) and b) in the program "Zenwriter" already. To be fair, it's a much less complex program tho.

c) Custom colors: Again - no idea how complicated to program.
Basically the user can choose between different preset (or freely choose and mix colors) color schemes.

**Comment by cameronsutter** [comment]
use: https://www.npmjs.com/package/react-custom-properties in the same way the creator did: https://medium.com/geckoboard-under-the-hood/how-we-made-our-product-more-personalized-with-css-variables-and-react-b29298fde608 to enable custom colors

Give a simple UI that let's you map names of things (background, tab color, etc) to color

Save that as a json file in their userData folder and load into the Redux store on bootup

**Comment by cameronsutter** [comment]
Color Picker
https://casesandberg.github.io/react-color
